### PR TITLE
Free weather related APIs

### PR DESCRIPTION
**Pull request by abhijeet007rocks8**
Added Weather API
- [WeatherBit](https://www.weatherbit.io/api)
- [WeatherAPI](https://www.weatherapi.com/docs/)

Related to #2 @TheLearneer

### CODE REVIEW

File: src/content/apis/weather/weatherAPI.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Weather API
+description: It is a powerful fully managed free weather and geolocation API provider that provides extensive APIs that range from the weather forecast, historical weather, future weather, weather alerts, air quality data, IP lookup, and astronomy through to sports, time zone, and geolocation.
+categories:
+  - weather
+features:
+  https: true
+url: https://www.weatherapi.com/docs/
+logo: https://cdn.weatherapi.com/v4/images/weatherapi_logo.png
```
Comment (TheLearneer) [inline src/content/apis/weather/weatherAPI.md:9]: The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.

File: src/content/apis/weather/weatherAPI.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Weather API
+description: It is a powerful fully managed free weather and geolocation API provider that provides extensive APIs that range from the weather forecast, historical weather, future weather, weather alerts, air quality data, IP lookup, and astronomy through to sports, time zone, and geolocation.
```
Comment (TheLearneer) [inline src/content/apis/weather/weatherAPI.md:3]: Maybe shorten this description a bit, seems a lot 😅

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Hi @abhijeet007rocks8 
Please go through the comments 🙏

**Comment by abhijeet007rocks8** [comment]
@TheLearneer Requsted changes are made

**Pull request merged**